<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monthScheduleCheck</class>
 <widget class="QWidget" name="monthScheduleCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>650</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>480</width>
      <height>40</height>
     </rect>
    </property>
    <widget class="QPushButton" name="exit_Button">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>YearMonthDay</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>10</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>480</width>
      <height>10</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>460</width>
      <height>590</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>10</width>
      <height>590</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>10</width>
      <height>590</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
